ups/base.cfg

```diff
@@ -3,8 +3,7 @@
 import lsst.sconsUtils
 
 dependencies = {
-    "required": ["boost"],
-    "buildRequired": ["swig"],
+    "required": ["boost", "pybind11"],
```

Comment:
`pybind11` is still just a build-time dependency, right, since it's header-only?

I don't think we actually use the distinction anywhere, but it'd be good to maintain it just in case.